svgnative/test/paint.svg

```diff
@@ -42,12 +42,13 @@
     </g>
     <rect x="140" y="40" width="20" height="20" fill="#F00 red" stroke="#F00 red"/>
     <rect x="160" y="40" width="20" height="20" fill="#F00 #F00" stroke="#F00 #F00"/>
-    <rect x="180" y="40" width="20" height="20" fill="url(#notPresent) #008000" stroke="url(#notPresent) #008000"/>
-    <rect x="0" y="60" width="20" height="20" fill="url(#notPresent) green" stroke="url(#notPresent) green"/>
+    <rect x="180" y="40" width="20" height="20" fill="url(#notPresent) #008000" stroke=" url(#notPresent) #008000"/>
+    <rect x="0" y="60" width="20" height="20" fill=" url(#notPresent) green" stroke="url(#notPresent) green"/>
```

Comment:
Is this spacing change relevant?

Reply:
Yes, it is testing the improvement of the refactoring+bug fixing.